yarn-project/simulator/src/client/private_execution.ts

```diff
@@ -46,9 +46,7 @@ export async function executePrivateFunction(
   const encryptedLogs = context.getEncryptedLogs();
   const unencryptedLogs = context.getUnencryptedLogs();
   // TODO(https://github.com/AztecProtocol/aztec-packages/issues/1165) --> set this in Noir
-  publicInputs.encryptedLogsHash = Fr.fromBuffer(encryptedLogs.hash());
   publicInputs.encryptedLogPreimagesLength = new Fr(encryptedLogs.getSerializedLength());
```

Comment:
We don't have to reassign logPreimageLengths either!

Reply:
I don't think they are ever pushed to the 'real' public inputs as we only push hashes without lengths - it's part of #1165 to actually get the lengths and add to `PrivateContext`